### PR TITLE
Add support for simpler "Table component" classes

### PR DESCRIPTION
**Pull request by phndiaye**
### What does this PR do?

Set of classes to build a simple in replacement of the upf-table components. This is just to build small tables that are not supposed to have advanced features like columns management, search, or filtering.

Related to: https://github.com/upfluence/backlog/issues/1714

### What are the observable changes?

Screenshot 2022-07-12 at 14 24 06

### Good PR checklist

- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- [ ] Added/updated tests
- [x] Added/updated documentation
- [ ] Migrated touched components to Glimmer Components
- [x] Properly labeled

### CODE REVIEW

File: addon/components/upf-table/pagination.hbs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<span class="text-size-5 margin-left-px-12">{{this.from}} to {{this.to}} out of {{this.itemTotal}} {{this.itemName}}</span>
```
Comment (Miexil) [inline addon/components/upf-table/pagination.hbs:1]: intl for "to" & "out of" ? 🤔

File: app/styles/organisms/Table.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,126 @@
+import { Meta } from '@storybook/addon-docs/blocks';
```
Comment (Miexil) [inline app/styles/organisms/Table.stories.mdx:1]: filename has a capital letter! 😱

**Review by Miexil — APPROVED** [review]

File: addon/components/upf-table/pagination.hbs [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<span class="text-size-5 margin-left-px-12">
+  {{t  "oss-components.upf-table.pagination" from=this.from to=this.to total=this.itemTotal item_name=this.itemName}}
```
Comment (JulienVannier66) [inline addon/components/upf-table/pagination.hbs:2]:
```suggestion
  {{t "oss-components.upf-table.pagination" from=this.from to=this.to total=this.itemTotal item_name=this.itemName}}
```
extra space

**Review by JulienVannier66 — APPROVED** [review]
Otherwise, LGTM

**Pull request merged**